working/0516 - strict-raw-types/strict-raw-types.md

```diff
@@ -0,0 +1,125 @@
+# Strict raw types static analysis option
+
+This document specifies the "Strict raw types" mode enabled with a static
+analysis option. As a static analysis option, we only intend to implement this
+feature in the Dart Analyzer. Under this feature, a type with omitted type
+argument(s) is defined as a "raw type." Dart fills in such type arguments with
+their bounds, or `dynamic` if there are no bounds.
+
+## Enabling strict raw types
+
+ To enable strict raw types, set the `strict-raw-types` option to `true`, under
+ the Analyzer's `language` section:
```

Comment:
Is there a command line option version of this flag?

Reply:
No, there isn't presently. I can't find the review comment, but I think @bwilkerson recommended just wiring an analysis option, because we're primarily dogfooding this internally, where it will exclusively be wired via an analysis options file.

 I don't think there was an intention to never add a command-line flag, it was just simpler not to for now.